walk.go

```diff
@@ -356,3 +362,15 @@ func signatureEntries(root string) []Entry {
 
 	return sigEntries
 }
+
+// keywordEntries returns a slice of entries including a comment of the
+// keywords requested when generating this manifest.
+func keywordEntries(keywords []Keyword) []Entry {
+	// Convert all of the keywords to zero-value keyvals.
+	return []Entry{
+		{
+			Type: CommentType,
+			Raw:  fmt.Sprintf("#%16s%s", "keywords: ", strings.Join(FromKeywords(keywords), ",")),
```

Comment:
Wait, I'm confused. How does that solve the problem? We're not parsing this (the rest of my code made sure it was parsed). Though as you said you're doing some clean-up of concepts so I'm sure you'll figure out a nicer way of solving it.